src/Main.js

```diff
@@ -74,12 +74,31 @@ WebhookHandler.on('push', HandlerWrap((ev) => {
     }
 }));
 
+// prUrl format, e.g., https://api.github.com/repos/github/hello-world/pulls/1
```

Comment:
Please move this useful URL format comment inside the function.

Reply:
Moved.